airsonic-main/src/main/java/org/airsonic/player/service/search/SearchServiceUtilities.java

```diff
@@ -71,6 +79,47 @@
     @Autowired
     private MediaFileService mediaFileService;
 
+    private static final Logger LOG = LoggerFactory.getLogger(SearchServiceUtilities.class);
+
+    private Random random;
+
+    {
+
+        String algorithmNative = "NativePRNG";
+        String algorithmSHA = "SHA1PRNG";
+
+        List<String> algorithms = Arrays.asList(Security.getProviders()).stream()
+                .flatMap(p -> p.getServices().stream().filter(s -> "SecureRandom".equals(s.getType())))
+                .map(s -> s.getAlgorithm()).collect(Collectors.toList());
+
+        if (algorithms.contains(algorithmNative)) {
+            /* linux */
+            try {
+                random = SecureRandom.getInstance(algorithmNative);
+                LOG.info("NativePRNG is used to create a random list of songs.", algorithmNative);
+            } catch (NoSuchAlgorithmException e) {
+                LOG.error("Usually unreachable.", e);
+            }
+        } else if (algorithms.contains(algorithmSHA)) {
+            /* win */
```

Comment:
Minor nit: I would spell out windows, makes it easier to grep for without false positives
Also for these single line comments, I suggest using single line `//` for comments that don't span lines